docs/rhoas/rhoas-cli-installation/README.adoc

```diff
@@ -257,6 +258,9 @@ $ sudo mv rhoas_completions /etc/bash_completion.d/rhoas
 ----
 $ sudo mv rhoas_completions /usr/local/etc/bash_completion.d/rhoas
 ----
+
+NOTE: For macOS consult your bash software for the location and usage of your bash_completion directory.
```

Comment:
If the user's bash-completion is in a different location, I believe they can find it by checking .bashrc or .bash_profile in their home directory:

```suggestion
NOTE: For macOS, to find the location of your `bash-completion` directory, check the `~/.bashrc` or `~/.bash_profile` file.
```

Reply:
Implemented, thanks